FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -176,14 +176,15 @@ extension EncryptedStorage {
     }
 
     func getKeypairs(by email: String) throws -> [Keypair] {
-        return try storage.objects(KeypairRealmObject.self).where({
-            $0.account == email
+        // Can't ues where because `where` doesn't support throws
```

Comment:
What about using `user.email` directly, like this:
```swift
return try storage.objects(KeypairRealmObject.self).where({
    $0.user.email == email
})
```

Reply:
Yeah. In that way,  that's fine. But `account` variable declaration becomes useless then.

Reply:
Then I think we can remove `account` variable, as it just adds more complexity to code, and use `user.email` directly.

Reply:
what do you suggest?

Reply:
Ok, I agree